### ISSUE TITLE
[Design] Create design for eFolders documents page 

### ISSUE THREAD

**Issue by leahkeeler** [department-of-veterans-affairs/va.gov-team#12375]
## Issue Description
Design page that aligns with design library and style guides at va.gov that replicates the functionality of page on eBenefits 

---
## Tasks
- [ ] Document listing page 
![image.png](https://images.zenhubusercontent.com/5e41d853bb7b6d67d84e6080/76aa2e6b-8627-47e9-9d76-733a487ee4f3)
- [ ] Search box for document type 
![image.png](https://images.zenhubusercontent.com/5e41d853bb7b6d67d84e6080/413a2e2a-8563-4cd0-9d93-5dd9ba68f4e7)
- [ ] Search box by date 
![image.png](https://images.zenhubusercontent.com/5e41d853bb7b6d67d84e6080/da1ab276-7768-4c57-b3c1-e3ff90c4cfd5)

## Acceptance Criteria
- [ ] _Design is complete and ready for developers to code and design intent review 
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by rileyorr** [comment]
Grabbed some additional screenshots and started adding notes to [this google doc](https://docs.google.com/document/d/1VaBsBx771AAx2lIkC7xlpIpjaBbhTEISR5RYOKAPvOc/edit?usp=sharing)

**Comment by rileyorr** [comment]
I've finished [the design's general layout](https://preview.uxpin.com/871719568f97443d6bc1b6ecd27180cf7c2cdf6b#/pages/131882489/simulate/no-panels?mode=i), starting with mobile first! I've reached out about the calendar picker component; I'll add that in once I get more info. Also, I'm going to iterate on the dropdown search component because I just have a general idea down atm. I'll create separate tickets for those items :)
